cloudferry_devlab/Vagrantfile

```diff
@@ -1,209 +1,162 @@
+  unless value.nil?
+    value = value.strip
     options[option] = value
   end
-}
+end
 
 nodes = {
-  "grizzly" => {
-    "box" => "openstack-user/precise-grizzly",
-    "ip" => "#{options['grizzly_ip']}",
-    "ip2" => "#{options['grizzly_ip2']}",
-    "memory" => 4096,
-    "role" => "openstack",
-    "cpus" => 2,
-    "nfs_share_name" => "grizzly"
-  },
-  "icehouse" => {
-    "box" => "openstack-user/trusty-icehouse",
-    "ip" => "#{options['icehouse_ip']}",
-    "ip2" => "#{options['icehouse_ip2']}",
-    "memory" => 4096,
-    "role" => "openstack",
-    "cpus" => 2,
-    "nfs_share_name" => "icehouse"
-  },
-  "juno" => {
-    "box" => "openstack-user/trusty-juno",
-    "ip" => "#{options['juno_ip']}",
-    "ip2" => "#{options['juno_ip2']}",
-    "memory" => 4096,
-    "role" => "openstack",
-    "cpus" => 2,
-    "nfs_share_name" => "juno"
+  'grizzly' => {
+    'box' => 'latest_grizzly',
+    'url' => 'http://172.18.236.71/clean_boxes/latest_grizzly.box',
+    'ip' => options['grizzly_ip'],
+    'ip2' => options['grizzly_ip2'],
+    'memory' => 4096,
+    'role' => 'openstack',
+    'cpus' => 2
   },
-  "cloudferry" => {
-    "box" => "hashicorp/precise64",
-    "ip" => "#{options['cloudferry_ip']}",
-    "ip2" => "#{options['cloudferry_ip2']}",
-    "memory" => 2048,
-    "role" => "lab"
+  'icehouse' => {
+    'box' => 'latest_icehouse',
+    'url' => 'http://172.18.236.71/clean_boxes/latest_icehouse.box',
+    'ip' => options['icehouse_ip'],
+    'ip2' => options['icehouse_ip2'],
+    'memory' => 4096,
+    'role' => 'openstack',
+    'cpus' => 2
   },
-  "nfs" => {
-    "box" => "openstack-user/nfs-server",
-    "ip" => "#{options['nfs_ip']}",
-    "ip2" => "#{options['nfs_ip2']}",
-    "memory" => 1024,
-    "cpus" => 1
+  'juno' => {
+    'box' => 'latest_juno',
+    'url' => 'http://172.18.236.71/clean_boxes/latest_juno.box',
+    'ip' => options['juno_ip'],
+    'ip2' => options['juno_ip2'],
+    'memory' => 4096,
+    'role' => 'openstack',
+    'cpus' => 2
   },
-  "grizzly_juno_src" => {
-    "box" => "grizzly_juno/src",
-    "url" => "http://172.18.236.71/grizzly_juno/src.json",
-    "ip" => "#{options['grizzly_ip']}",
-    "ip2" => "#{options['grizzly_ip2']}",
-    "memory" => 4096,
-    "role" => "openstack",
-    "hostname" => "grizzly",
-    "release" => "grizzly",
-    "cpus" => 2,
-    "nfs_share_name" => "grizzly"
+  'nfs' => {
+    'box' => 'latest_nfs',
+    'url' => 'http://172.18.236.71/clean_boxes/latest_nfs.box',
+    'ip' => options['nfs_ip'],
+    'ip2' => options['nfs_ip2'],
+    'memory' => 1024,
+    'cpus' => 1
   },
-  "grizzly_juno_dst" => {
-    "box" => "grizzly_juno/dst",
-    "url" => "http://172.18.236.71/grizzly_juno/dst.json",
-    "ip" => "#{options['juno_ip']}",
-    "ip2" => "#{options['juno_ip2']}",
-    "memory" => 4096,
-    "role" => "openstack",
-    "hostname" => "juno",
-    "release" => "juno",
-    "cpus" => 2,
-    "nfs_share_name" => "juno"
+  'grizzly_juno_src' => {
```

Comment:
`grizzly_juno_src`, `grizzly_juno_dst`, `grizzly_juno_nfs`, `icehouse_juno_src`, `icehouse_juno_dst`, `icehouse_juno_nfs` boxes can be removed, they won't work with provided URLs anyway.